test/gtest/smith_for_loop_test.cpp

```diff
@@ -0,0 +1,81 @@
+#include "backends/p4tools/modules/smith/common/declarations.h"
+#include "backends/p4tools/modules/smith/common/expressions.h"
+#include "backends/p4tools/modules/smith/common/generator.h"
+#include "backends/p4tools/modules/smith/common/parser.h"
+#include "backends/p4tools/modules/smith/common/probabilities.h"
+#include "backends/p4tools/modules/smith/common/scope.h"
+#include "backends/p4tools/modules/smith/common/statements.h"
+#include "backends/p4tools/modules/smith/common/table.h"
+#include "backends/p4tools/modules/smith/core/target.h"
+#include "backends/p4tools/modules/smith/smith.h"
+#include "gtest/gtest.h"
+#include "ir/ir-generated.h"
```

Comment:
```suggestion
```

Reply:
> 

Change applied.

Reply:
You do not need to reply to every comment, this creates a lot of email traffic. Trivial ones like this one you can just "resolve".

Reply:
> You do not need to reply to every comment, this creates a lot of email traffic. Trivial ones like this one you can just "resolve".

Sorry, I will do that in the future!